src/connectionManager.ts

```diff
@@ -0,0 +1,84 @@
+// TODO: Reference a library instead for GUID generation?
+export type GUID = string & { isGuid: true };
+export function guid(guid: string): GUID {
+  return guid as GUID;
+}
+
+export function generateGUID(): string {
```

Comment:
Why does this return a `string` and not a `GUID`?

Reply:
Replaced with uuid library.